testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -5859,10 +5859,10 @@ private static void checkIf(SqlOperatorFixture f) {
     f.checkString("CURRENT_CATALOG", "", "VARCHAR(2000) NOT NULL");
   }
 
-  @Tag("slow")
-  @Test void testLocalTimeFuncWithCurrentTime() {
-    testLocalTimeFunc(currentTimeString(LOCAL_TZ));
-  }
+//  @Tag("slow")
```

Comment:
is this meant to be commented out?

Reply:
oh lol oops, commented out cause it was slow when running tests, nice catch!